python/mp/effects/bounce.py

```diff
@@ -14,16 +14,15 @@ class Bounce(effect.Effect):
     # Wish there were a better way than requiring this every time
     dm = copy.deepcopy(effect.Effect.dm)
 
-    def __init__(self, bead_set, color=color.Color(), duration=None, direction=1):
-        super().__init__("bounce", bead_set, color=color, duration=duration)
+    def __init__(self, bead_set, color=color.Color(), direction=1, **kwargs):
+        super().__init__(name="bounce", bead_set=bead_set, color=color, **kwargs)
```

Comment:
IMPORTANT NOTE! All args now have to be passed to super as keyword arguments, and no longer positional arguments. If you look in effect.py later, you'll see why.